ivadomed/models.py

```diff
@@ -961,6 +962,129 @@ def forward(self, inputs):
         return outputs
 
 
+class ConvBlock(nn.Module):
+    def __init__(self, in_chan, out_chan, ksize=3, stride=1, pad=0, activation=nn.LeakyReLU()):
+        """
+        Perform convolution, activation and batch normalization.
+        Args:
+            in_chan (int): number of channels on input
+            out_chan (int): number of channel on output
+            ksize (int): size of kernel for the 2d convolution
+            stride (int): strides for 2d convolution
+            pad (int): pad for nn.conv2d
+            activation (nn.layers): activation layer. default Leaky ReLu
+        """
+        super(ConvBlock, self).__init__()
+        self.conv1 = nn.Conv2d(in_chan, out_chan, kernel_size=ksize, stride=stride, padding=pad)
+        self.activation = activation
+        self.batch_norm = nn.BatchNorm2d(out_chan)
+
+    def forward(self, x):
+        return self.activation(self.batch_norm(self.conv1(x)))
+
+
+class SimpleBlock(nn.Module):
+    def __init__(self, in_chan, out_chan_1x1, out_chan_3x3, activation=nn.LeakyReLU()):
+        """
+        Inception module with 3 convolutions with different kernel size with separate activation.
+        The 3 outputs are then concatenated. Max pooling performed on concatenation.
+        Args:
+            in_chan (int): number of channel of input
+            out_chan_1x1 (int): number of channel after first convolution block
+            out_chan_3x3 (int): number of channel for the other convolution blocks
+            activation (nn.layers): activation layer used in convolution block
+        """
+        super(SimpleBlock, self).__init__()
+        self.conv1 = ConvBlock(in_chan, out_chan_1x1, ksize=3, pad=0, activation=activation)
+        self.conv2 = ConvBlock(in_chan, out_chan_3x3, ksize=5, pad=1, activation=activation)
+        self.conv3 = ConvBlock(in_chan, out_chan_3x3, ksize=9, pad=3, activation=activation)
+        self.MP = nn.MaxPool2d(1)
+
+    def forward(self, x):
+        conv1_out = self.conv1(x)
+        conv2_out = self.conv2(x)
+        conv3_out = self.conv3(x)
+        output = torch.cat([conv1_out, conv2_out, conv3_out], 1)
+        output = self.MP(output)
+        return output
+
+
+class Countception(nn.Module):
+    """Countception model
+
+    .. seealso::
+        Paul Cohen, Joseph, et al. "Count-ception: Counting by fully convolutional redundant counting."
```

Comment:
"Cohen" is the name, "Joseph Paul" are first names. Just put the initials 
```suggestion
        Cohen JP et al. "Count-ception: Counting by fully convolutional redundant counting."
```